tests/AssertTest.php

```diff
@@ -729,6 +729,27 @@ public function testAnUnknownMethodThrowsABadMethodCall()
 
         Assert::nonExistentMethod();
     }
+
+    public function getInvalidIsAOfCases()
+    {
+        return array(
```

Comment:
Let's `yield` this instead, that is easier to read.

Reply:
Done